src/code-a-long/css-selectors/form.html/form.html

```diff
@@ -0,0 +1,56 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>From</title>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+  <div class="main-container">
+    <div class="form-container">
+      <div class="form-container--img"></div>
+        <div class="form-container--form">
+          <div class="form-inner">
+          <h1>REGISTER YOUR TRIP</h1>
+          <div class="input-group">
+          <label for="name">Name</label>
+          <input id="name" type="text">
+        </div>
+        <div class="input-group">
+          <div class="inner-group">
+            <label for="date">Date</label>
+            <input type="date">
+          </div>
+        </div>
+        <div>
+          <div class="inner-group">
+            <label for="gender">Gender</label>
+            <select name="gender" id="gender">
+              <option value="male">Male</option>
+              <option value="female">Female</option>
+              <option value="unicorn ">Unicorn</option>
+            </select>
+          </div>
+        </div>
+        <div class="input-group class">
+            <label for="gender">Destination</label>
+            <select name="destination" id="destination">
+              <option value="male">Telluride</option>
+              <option value="female">Breckenridge</option>
+              <option value="unicorn ">Durango</option>
+            </select>
+
+        </div>
+        <button>Submit</button>
+      
+        </div>
+
+    </div>
+  </div>
+
+
+  </div>
```

Comment:
your html files are looking better! clean! and i can find things! 🔥. tighten it up though. no extra spaces. annoying, i know!